R/read_and_prepare.R

```diff
@@ -79,16 +79,16 @@ read_and_prepare_project_agnostic_data <- function(start_year, end_year, company
 #' @inheritParams read_and_prepare_project_agnostic_data
 #' @inheritParams wrangle_and_check_pacta_results
 #' @param calculation_level String holding level of calculation.
+#' @param path String holding path to data.
 #'
 #' @return A list of lists holding prepared project specific data.
 read_and_prepare_project_specific_data <- function(asset_type, calculation_level,
                                                    time_horizon, scenario_geography_filter,
                                                    scenarios_filter, equity_market_filter,
-                                                   term) {
-  path <- file.path(get_st_data_path("ST_PROJECT_FOLDER"), "inputs", paste0(stringr::str_to_title(asset_type), "_results_", calculation_level, ".rda"))
+                                                   term, path) {
 
   pacta_results <- read_pacta_results(
-    path = path,
+    path = file.path(path, "inputs", paste0(stringr::str_to_title(asset_type), "_results_", calculation_level, ".rda")),
```

Comment:
An alternative:

``` r
stringr::str_to_title(c("bonds", "loans"))
#> [1] "Bonds" "Loans"
tools::toTitleCase(c("bonds", "loans"))
#> [1] "Bonds" "Loans"
```

<sup>Created on 2021-11-10 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>